extensions/WEBGL_color_buffer_float/extension.xml

```diff
@@ -38,13 +38,14 @@
       </feature>
 
       <feature>
-        <p>The 32-bit floating-point type <code>RGB32F</code> may become available
-        as a color-renderable format. Renderbuffers can be created in this
-        format. These and textures created with <code>format = RGB</code> and
-        <code>type = FLOAT</code> as specified in <a
+        <p>The 32-bit floating-point type <code>RGB32F</code> may also optionally
+	become available as a color-renderable format. Renderbuffers can be created
```

Comment:
Renderbuffers cannot be created in this format. Just remove that line from here.

Reply:
Removed